code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1503,10 +1502,14 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(target.IsKnockdown())
 			var/target_held_item = target.get_active_held_item()
 			if(target_held_item)
-				target.drop_all_held_items()
 				target.visible_message("<span class='danger'>[user.name] kicks \the [target_held_item] out of [target]'s hand!</span>",
 									"<span class='danger'>[user.name] kicks \the [target_held_item] out of your hand!</span>", null, COMBAT_MESSAGE_RANGE)
 				log_combat(user, target, "disarms [target_held_item]")
+			else
+				target.visible_message("<span class='danger'>[user.name] kicks [target.name] onto [target.p_their()] side!</span>",
+									"<span class='danger'>[user.name] kicks you onto your side!</span>", null, COMBAT_MESSAGE_RANGE)
+				log_combat(user, target, "kicks", "onto their side (paralyzing)")
```

Comment:
This should honestly just be the same message and log

Reply:
I think it's extremely relevant to log whether a weapon was actually disarmed or it was just a stun. There were many *many* cases of "But they pulled out a weapon" that I had no logs to verify for because the supposed wasn't logged anywhere else. 

Would request you get admin input on whether the difference should be logged for them or not.

Reply:
This won't log inactive held items though